.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org/
```

Comment:
This is precisely what we're trying to avoid. If this conf is missing, packages regularly mention private registry urls that are inaccessible to the rest of the community. This is in place on the tock studio repo and has solved the recurring problems that existed previously.

Reply:
I see, that makes sense.